data/scenarios/Tutorials/craft.yaml

```diff
@@ -9,12 +9,12 @@ description: |
 objectives:
   - goal:
       - Robots can use the `make` command to make things, as long as
-        they have a `workbench` and the proper ingredients.  For
+        they have a `workbench`{=entity} and the proper ingredients.  For
         example, `make "circuit"` will make a circuit.
       - Your base has a few trees in its inventory. Select them to see the
         available recipes.
-      - Your goal is to make a "branch predictor", so you will have to make
-        some "branch"es first.
+      - Your goal is to make a `branch predictor`{=entity}, so you will have to make
+        some `"branch"`es first.
```

Comment:
Why not
```suggestion
        some `branch`{=entity}es first.
```
?

Reply:
Because `"branch"` is what the player should write and it is valid swarm expression.

I have considered displaying all entities with quotes, but that could be misleading if written without them.

Either way, with the `{=entity}` tag we can change our mind and replace all occurrences using regex. 🙂